lib/gssapi/sanon/negoex.c

```diff
@@ -64,7 +64,7 @@ _gss_sanon_inquire_negoex_key(OM_uint32 *minor,
 	return GSS_S_UNAVAILABLE;
     }
 
-    initiator_key = !!(sc->flags & SANON_FLAG_INITIATOR);
+    initiator_key = !!(sc->is_initiator);
```

Comment:
the `!!` is redundant, can be removed

Reply:
Meh.

Reply:
I think it would be cleaner to following if it was just `sc->is_initiator`